packages/dcos-integration-test/extra/integration_test.py

```diff
@@ -1105,6 +1105,79 @@ def _ensure_routable():
     _ensure_routable()
 
 
+def test_if_minuteman_routes_to_named_vip(cluster, timeout=125):
```

Comment:
we never will change this timeout / configure it and it's only referenced one place, just make it live in one place.

2 minutesis also wayyy longer than this should take.